solver/src/http_solver.rs

```diff
@@ -23,9 +24,11 @@ use std::{
     sync::Arc,
 };
 
+const GAS_PER_ORDER: u128 = 37676;
```

Comment:
Maybe add a comment on how you derived at the values. I feel they are a bit low, given that a settlement with one trade and Uniswap already costs > 180k gas.

Reply:
Done. For "better" estimations we should probably use a table close to the values we are expecting (I guess we aren't expecting 50 trades).